Code.gs

```diff
@@ -59,6 +75,31 @@ function onGmailMessage( event ) {
   return buildCustomerCard( emailAddress, orderDataForEmailAddress );
 }
 
+/**
+ * Handle the email search from the homepage.
+ *
+ * @param {Object} e Event object containing form inputs.
+ * @returns {CardService.Card}
+ */
+function onSearchByEmail(e) {
+  const emailAddress = e.formInput.searchEmail;
```

Comment:
Should we make sure this path is fully defined before accessing it?
```suggestion
  if ( ! e || ! e.formInput ) {
    // return something useful!
  }
  const emailAddress = e.formInput.searchEmail;
```

Reply:
Thanks Dale 👍🏼  No rush at all, I'm using the PR in my inbox already 💪🏼